english/normal-dungeon/3/2.md

```diff
@@ -0,0 +1,5 @@
+As you approach your home, the 🚪door is ajar, ever so slightly, 🕯candlight emanating from the cracks.
```

Comment:
Hi @aUsefulMethod, nice story but the options are not pointing to other rooms

Reply:
I was going to leave those open if anyone wanted to hop in and create the next part (sub-rooms). I can fix that and make them point to existing rooms if you want?

Reply:
Yes

Reply:
Ok, I added fix :)

Reply:
Nice work 👍